website/next.config.mjs

```diff
@@ -13,43 +8,52 @@ const env = {
     process.env.ENVIRONMENT === 'production' ? 'cfeac8baf33c9b4d255f28d57f3c9148' : 'e57a9892339b2acfd02943c86b746d32',
 }
 
-const withMDX = mdx({
-  extension: /\.mdx?$/,
-  options: {
-    remarkPlugins: [remarkGfm, remarkFrontmatter, [remarkMdxFrontmatter, { name: 'frontmatter' }], remarkMdxLayout],
-    providerImportSource: '@mdx-js/react',
-    jsxImportSource: 'theme-ui',
+const withNextra = nextra({
+  theme: './layout/index.ts',
+  flexsearch: false,
+  staticImage: true,
+  codeHighlight: false,
+  // defaultShowCopyCode: true,
+  transform(content, { route }) {
+    if (route) {
+      return `
+import { getNavItems } from '@/navigation'
+
+${content}
+
+export const getStaticProps = async () => {
+  const navItems = await getNavItems('${route.split('/')[1]}')
+  return {
+    props: { navItems }
+  }
+}`
```

Comment:
the temporal solution, once we'll use `_meta.json` I'll remove it and we'll have the possibility that every page could have their`getStaticProps`